src/vmm/src/devices/virtio/mod.rs

```diff
@@ -68,6 +70,8 @@ pub enum ActivateError {
     BadActivate,
     /// Vhost user: {0}
     VhostUser(vhost_user::VhostUserError),
+    /// Setting tap interface offload flags failed
+    TapSetOffload(TapError),
```

Comment:
Use `{0}` (or `{0:?}` if `TapError` does not implement `Display` trait) at the end of the comment. We used special magic to make comments act as Display implementations for errors.